plugins/Marketplace/tests/UI/ManageLicense_spec.js

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * Screenshot tests for Marketplace.
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("ManageLicense", function () {
+  this.timeout(0);
+
+  this.fixture = "Piwik\\Plugins\\Marketplace\\tests\\Fixtures\\SimpleFixtureTrackFewVisits";
+
+  const urlBase = '?module=Marketplace&action=manageLicenseKey';
+
+  async function captureSelector(screenshotName, selector)
+  {
+    await page.evaluate(function () {
```

Comment:
The menu isn't used in any of the tests and it will be resetted by reloading the page. But hope we can remove that again, once we have a final solution how to handle that menu shadow properly.